scripts/collect_images/kelly.py

```diff
@@ -0,0 +1,167 @@
+from dotenv import load_dotenv
+import os
+import requests
+import hashlib
+import hmac
+import base64
+import urllib.parse as urlparse
+import matplotlib.pyplot as plt
+import matplotlib.image as mpimg
+
+# Load the .env file
+load_dotenv()
+# Call the variables in the .env file
+api_key = os.getenv("API_KEY")
+sign_key = os.getenv("SIGN_KEY")
```

Comment:
Code worked to generate images. Should change "SIGN_KEY" in .env file and argument for os.getenv() to "SECRET" to be consistent with other scripts.